#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_support_for_lzma.diff
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: support LZMA compression at squashfs

@DPATCH@

--- a/hooks/live  2008-07-17 10:29:18.000000000 +0200
+++ b/hooks/live  2008-07-17 10:29:30.000000000 +0200
@@ -90,6 +90,8 @@
 copy_exec /sbin/losetup /sbin
 manual_add_modules loop
 manual_add_modules squashfs
+manual_add_modules sqlzma
+manual_add_modules unlzma

 # Filesystem: aufs/unionfs
 manual_add_modules aufs

